tests/std/tests/VSO_0157762_feature_test_macros/test.cpp

```diff
@@ -5,465 +5,431 @@
 
 int main() {} // COMPILE-ONLY
 
-// COMPILER FEATURE-TEST MACROS
+// ATTRIBUTE FEATURE-TEST MACROS.

```

Comment:
"Attribute feature-test macros" isn't a sentence, so it should not end with a period:
```suggestion
// ATTRIBUTE FEATURE-TEST MACROS
```